openedx/core/djangoapps/credentials/management/commands/notify_credentials.py

```diff
@@ -164,6 +169,9 @@ def handle(self, *args, **options):
             cert_filter_args['modified_date__lte'] = options['end_date']
             grade_filter_args['modified__lte'] = options['end_date']
 
+        if options['mode']:
+            cert_filter_args['mode'] = options['mode']
```

Comment:
We're not adding a filter for grades here too, but we should. Though it would be difficult to filter based on the mode of the related cert via just a Django query...

Alternatively, don't do any grades if we're given a mode option. Just do certs. (And warn user via flag documentation and maybe a print that that's happening?)